DynamicTablesPkg/DynamicTables.dsc.inc

```diff
@@ -67,6 +68,8 @@
       # Generators
       #   Common
       NULL|DynamicTablesPkg/Library/Acpi/Common/AcpiFadtLib/AcpiFadtLib.inf
+      NULL|DynamicTablesPkg/Library/Acpi/Common/AcpiSpmiLib/AcpiSpmiLib.inf
+      #   X64 specific

```

Comment:
@samimujawar the generator could also be added to the arm section right ?

Reply:
Yes, this would be used by Arm as well.